server/adaptors/integrations/__data__/repository/k8s/k8s-1.0.0.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "k8s",
+    "version": "1.0.0",
+    "displayName": "k8s Dashboard",
```

Comment:
Do we want to display it as "k8s" instead of "Kubernetes"? I don't know the ecosystem well, it just seems a tad informal

Reply:
yes